### PR TITLE
feat: describe chargeback statement line items and usage vs. chargeba…

### PR DESCRIPTION
**Pull request by JohannesRudolph**
…ck periods

these terms clarify what data the CSV export is showing vs. what's
displayed in meshPanel

### CODE REVIEW

File: docs/meshcloud.project-metering.md [hunk 1]
```diff
@@ -45,11 +50,16 @@ Therefore, the unit costs are not available in the Tenant Usage Reports for thos
...
+usage reports that were entered before the end of the chargeback period and have not been included in a previous
+chargeback statement.
+
+> meshStack defines the **chargeback period** as a monthly period with a configurable offset (e.g. 6 days)
+> allowing cloud providers to finish processing of usage reports for the usage period.
 
 As an anology from banking you can think of a chargeback account like a credit account and the chargeback statement like
-a credit card statement you receive at the end of the month about all transactions charged to the card.
+a credit card statement you receive at the end of the month about all transactions charged to the card. Credit card
+statements do as well show charges with a booking and valuta date.
```
Comment (Prerna21122112) [inline docs/meshcloud.project-metering.md:62]: and valuta date.... what do you mean here...? valid?

File: docs/meshcloud.project-metering.md [hunk 2]
```diff
@@ -84,8 +94,21 @@ period. This preview is periodically updated and contains only reports available
...
-Chargeback Statements also contain billing information per line item. It can be [configured](meshstack.billing.md#chargeback) per meshImplementation which information shall be shown there. This info will occur in the UI when looking at the line items of a chargeback statements.
+
+#### Exporting Chargeback Statements
+
+It is possible to do an export from these views by clicking on the "CSV Export" button.
+This export will contain the line items (see above) of all the chargeback statements currently in the view.
+The line item data is suitable for feeding into chargeback processing, e.g. importing it to an ERP System to transfer
+budgets between cost centers.
+
+Chargeback Statements also contain billing information per line item. You Cloud Foundation team can [configure](meshstack.billing.md#chargeback)
```
Comment (Prerna21122112) [inline docs/meshcloud.project-metering.md:112]: Your Cloud Foundation team

File: docs/meshcloud.project-metering.md [hunk 3]
```diff
@@ -84,8 +94,21 @@ period. This preview is periodically updated and contains only reports available
...
+which information shall be shown there. This info will occur in the UI when looking at the line items of a chargeback statements.
```
Comment (Prerna21122112) [inline docs/meshcloud.project-metering.md:113]: this information will appear

File: docs/meshcloud.project-metering.md [hunk 4]
```diff
@@ -30,11 +30,16 @@ Additionally an amount can be set on the payment method, which indicates how muc
 
 A tenant usage report provides usage information of one tenant (project representation in a specific platform). E.g. all resources like VMs, Storage, Public IPs, etc are shown in this report.
```
Comment (Prerna21122112) [inline docs/meshcloud.project-metering.md:31]: For example, all resources....

File: docs/meshcloud.project-metering.md [hunk 5]
```diff
@@ -30,11 +30,16 @@ Additionally an amount can be set on the payment method, which indicates how muc
...
+Tenant Usage Reports can be accessed in the tenant control plane under **Finacials** > **Usage Reports**.
+meshStack generates a tenant usage report for every usage period (monthly).
+
+> meshStack defines **usage period** as starting from the first day of a month to the last day of the month in UTC.
+
+You also have access to a preview report of the current period (e.g. the current month), to always have full control of
 your resources and their usage.
 
-From a Tenant Usage Report you can also access a Detailed Report to see every single usage the resources created. That means you can e.g. see whether a resource was running for 3 hours, stopped for 5 hours and then started again for another 6 hours.
+From a Tenant Usage Report you can also access a Detailed Report to see every single usage the resources created.
```
Comment (Prerna21122112) [inline docs/meshcloud.project-metering.md:41]: From a Tenant Usage Report you can also access a Detailed Report to see every single usage the resources have created.

File: docs/meshcloud.project-metering.md [hunk 6]
```diff
@@ -84,8 +94,21 @@ period. This preview is periodically updated and contains only reports available
 
 A chart under **Financials** > **Chargeback Statements** in the project control plane shows the total amount charged as of the last chargeback statements to get a quick overview of your project cost.
 
+Chargeback statements consist of **line items** showing the individual booking entries made into a chargeback account.
+When booking a tenant usage report, meshStack splits up the report's `netAmount` by seller and product group into
```
Comment (Prerna21122112) [inline docs/meshcloud.project-metering.md:98]: When booking a tenant usage report, meshStack splits up the report's `netAmount` by seller and product group, into...

**Review by Prerna21122112 — APPROVED** [review]

**Review by Jelledb — APPROVED** [review]

**Pull request merged**